release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -100,17 +100,45 @@ For more information, see xref:../updating/preparing_for_updates/updating-cluste
 [id="ocp-4-16-web-console_{context}"]
 === Web console
 
+[id="ocp-4-16-web-console-language"]
+==== Language support for French and Spansih
+With this release, French and Spanish are suported in the web console. You can update the language in the webconsole from the *Language* list on the *User Preferences* page.
+
+[id="ocp-4-16-patternfly4-deprecated"]
+==== Patternfly 4 is now deprecated with 4.16
+With this release, Patternfly 4 and React Router 5 is deprecated in the web console. All plugins should migrate to Patternfly 5 and React Router 6 as soon as possible.
+
 [id="ocp-4-16-administrator-perspective_{context}"]
 ==== Administrator perspective
 
 This release introduces the following updates to the *Administrator* perspective of the web console:
-//Are there any for 4.16
+
+* A Google Cloud Platform (GCP) token authorization `Auth Token GCP`, and a `Configurable TLS ciphers` filter was added to the *Infrastructure features* filter in the OperatorHub.
+* A new quick start, *Build with guided documentation*, is available with information on impersonating the `system:admin` user.
+* A pod's last termination state is now available to view on the *Container list* and *Container details* pages.
+* An `Impersonate Group` action is now available from the *Groups* and *Group details* pages without having to search for the appropriate `RoleBinding`.
+
+[id="ocp-4-16-node-csr-handling"]
+===== Node CSR handling in the {product-title} web console
+With this release, the {product-title} web console supports node certificate signing requests (CSRs).
+
+[id="ocp-4-16-cross-storage-class-clone-restore"]
+===== Cross Storage Class clone and restore
+
+With this release, you can choose a storage class from the same provider when cloning or restoring. You can move from storage class with a replica of 3 to a replica of 2/1.
 
 [id="ocp-4-16-developer-perspective_{context}"]
 ==== Developer Perspective
 
 This release introduces the following updates to the *Developer* perspective of the web console:
-//* are there any for 4.16?
+
+* When searching, a new section was added to the list of *Resources* on the *Search* page to display the recently searched items in the order they were searched.
+* With this release, you can collapse and expand the *Getting started* section.
+
+[id="ocp-4-16-console-telemetry"]
+===== Console Telemetry
+
+With this release, we enabled user analytics with anonymized frontend telemetry if cluster telemetry is enabled. This is the default for most of the cluster and provides Red Hat with metrics for how the Console is used. Cluster administrators can update this in each cluster and opt-in, opt-out, or disable frontend telemetry.
```

Comment:
This sentence is correct.

I just thought that  "we enabled user analytics with anonymized frontend telemetry"  sounds "complicated".

Wdyt about this suggestion?

```suggestion
With this release, we enabled anonymized user analytics if cluster telemetry is enabled. This is the default for most of the cluster and provides Red Hat with metrics for how the Console is used. Cluster administrators can update this in each cluster and opt-in, opt-out, or disable frontend telemetry.
```

If you prefer your version that's also fine. :smirk:

Reply:
I like the suggestion! I was only trying to avoid using parenthesis since we only use them in specific cases.

Reply:
Updated